CHANGELOG.md

```diff
@@ -156,12 +168,24 @@ BUG FIXES:
 * ui: Disallow max versions value of large than 9999999999999999 on kv2 secrets engine. [[GH-9242](https://github.com/hashicorp/vault/pull/9242)]
 * ui: Add and upgrade missing dependencies to resolve a failure with `make static-dist`. [[GH-9277](https://github.com/hashicorp/vault/pull/9371)]
 
-## 1.4.6 (TBD)
+## 1.4.7 (TBD)
 
 IMPROVEMENTS:
 
 * secret/azure: Use write-ahead-logs to cleanup any orphaned Service Principals [[GH-9773](https://github.com/hashicorp/vault/pull/9773)]
```

Comment:
This should be in on 1.5.3 though: https://github.com/hashicorp/vault/pull/9769/files

Reply:
That feature debuted in 1.5.1

Reply:
Should I move it to 1.5.1 then?

Reply:
Yep, I'd say so. Jim verified that this was in 1.5.1.

Reply:
Yes, it should be in the 1.5.1 section (and only 1.5.1)